stdlib/stringLabels.mli

```diff
@@ -395,6 +395,48 @@ val is_valid_utf_16le : t -> bool
 (** [is_valid_utf_16le b] is [true] if and only if [b] contains valid
     UTF-16LE data. *)
 
+(** {1:spellchecking Spellchecking} *)
+
+val edit_distance : ?limit:int -> t -> t -> int
+(** [edit_distance s0 s1] is the number of single character edits
+    (understood as insertion, deletion, substitution, transposition)
+    that are needed to change [s0] into [s1].
+
+    The function assumes the strings are UTF-8 encoded and uses {!Uchar.t}
+    for the notion of character. Decoding errors are replaced by
+    {!Uchar.rep}. Normalizing the strings to
+    {{:https://unicode.org/glossary/#normalization_form_c}NFC} gives
+    better results.
```

Comment:
I would be tempted to move this paragraph towards the end of the documentation string (maybe after the note), because I suspect that for many it will be a point of detail relative to the meaning of the `~limit` parameter below.